x-pack/legacy/plugins/siem/public/components/skeleton_row/index.test.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import React from 'react';
+
+import { SkeletonRow } from './index';
+
+describe('rendering', () => {
+  test('renders correctly', () => {
```

Comment:
is some additional test setup necessary to verify correctness / behavior? the entirety of the snapshot output file appears to be just an empty tag:

```

// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `<Component />`;
```

Reply:
👍